ios/FluentUI/DrawerVnext/SlideOverPanel.swift

```diff
@@ -50,46 +103,38 @@ struct SlideOverPanel<Content: View>: View {
             }
 
             content
-                .frame(width: contentWidth())
-                .shadow(color: tokens.shadowColor.opacity(isOpen ? tokens.shadowOpacity : 0),
+                .frame(width: contentWidth)
+                .shadow(color: tokens.shadowColor.opacity(resolveTransitionOpacity(tokens.shadowOpacity)),
                         radius: tokens.shadowBlur,
                         x: tokens.shadowDepthX,
                         y: tokens.shadowDepthY)
-                .offset(x: resolvedContentOffset())
+                .offset(x: resolvedContentOffset)
 
             if direction == .left {
                 InteractiveSpacer(defaultBackgroundColor: $tokens.backgroundClearColor)
                     .onTapGesture (perform: actionOnBackgroundTap ?? {})
             }
         }
-        .background(isOpen ? tokens.backgroundDimmedColor.opacity(backgroundLayerOpacity) : tokens.backgroundClearColor)
+        .background(transitionState == .collapsed ? tokens.backgroundClearColor : tokens.backgroundDimmedColor.opacity(resolveTransitionOpacity(tokens.backgroundDimmedOpacity)))
     }
 
-    private func resolvedContentOffset() -> CGFloat {
-        // override offset if required
-        if let preferredContentWidth = preferredContentOffset {
-            // parent view wants to take over primarily to conform user drag gesture
-            return preferredContentWidth
+    private func resolveTransitionOpacity(_ maxValue: Double) -> Double {
+        let clearBackgroundOpacity = 0.0
+        guard backgroundDimmed else {
```

Comment:
This is always setting the shadow opacity to 0.
This check shouldn't be in this func because it only applies to the background.

How about creating a computed property to return the background color? You can check the backgroundDimmed and transition state in there and return the color with the opacity returned by this func.

Reply:
thanks for catching this, seems like I am almost numb to recurring issues, I will add some snapshot tests right after blocking issues are resolved